### ISSUE TITLE
supported / target browsers for helix pages

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-pages#44]
looking at current browser marketshare on:
http://gs.statcounter.com/browser-market-share/

i think that we should support the top 3 browsers for desktop and mobile, 

...as of May 2019:

Desktop
- Chrome - 69.09%
- Firefox - 10.01%
- Safari - 7.25%

Mobile 
- Chrome - 59.43%
- Safari - 20.81%
- Samsung Internet - 7.12%

i don't know that we would need to be very specific about the versions of these...

i am definitely open to supporting more browsers if we want to, but initially i don't think we need to make the list much bigger.

**Comment by rofe** [comment]
> i don't know that we would need to be very specific about the versions of these...

I don't think so. These are all evergreen browsers, so we always support the latest version.

**Comment by kptdobe** [comment]
This should be tackled as part of #8 and documented.
